apps.yaml

```
@@ -174,7 +174,7 @@ appsInfo:
     chartName: knative-operator
   kyverno:
     title: Kyverno
-    appVersion: 1.11.4
+    appVersion: 1.15.0
```

```suggestion
    appVersion: 1.11.4
```

Kyverno is not updated by this change, is it?